sdcm/argus_results.py

```diff
@@ -170,3 +170,16 @@ def _get_status_based_on_previous_results(metric: str):
     for key, value in stats.items():
         result_table.add_result(column=key, row="#1", value=value, status=_get_status_based_on_previous_results(key))
     argus_client.submit_results(result_table)
+
+
+def send_manager_benchmark_results_to_argus(argus_client: ArgusClient, result: dict, sut_timestamp: int):
+    class RestoreResult(GenericResultTable):
+        class Meta:
+            name = f"Restore duration"
+            description = "Restore duration"
+            Columns = [ColumnMetadata(name=param, unit="", type=ResultType.INTEGER) for param in result.keys()]
+
+    result_table = RestoreResult(sut_timestamp=sut_timestamp)
+    for key, value in result.items():
+        result_table.add_result(column=key, row="#1", value=value, status=Status.PASS)
```

Comment:
worth to add dataset size, e.g. instead of labeling row with `#1` could use `1TB` or snapshot name `1tb_1t_ics`

Reply:
Fixed